pkg/cimfs/mount_cim.go

```diff
@@ -63,3 +64,36 @@ func Unmount(volumePath string) error {
 
 	return nil
 }
+
+// MountMergedCims mounts the given merged CIM (usually created with `CreateMergedCim`) at the a volume with
+// given GUID. The `cimPaths` MUST be identical to the `cimPaths` passed to `CreateMergedCim` when creating
+// this merged CIM.
+func MountMergedCims(cimPaths []string, mergedCimPath string, mountFlags uint32, volumeGUID guid.GUID) (string, error) {
```

Comment:
My previous impression of merged cims was that each cim could be mounted at a specific subdir in the filesystem. Is that not the case?

Reply:
We won't need to explicitly mount individual CIMs, we just have to mount the final merged CIM.